scripts/whisper/rknn/test_on_rk3588_board.py

```
@@ -0,0 +1,343 @@
+        n_text_layer=n_text_layer,
+        n_text_ctx=n_text_ctx,
+        n_text_state=n_text_state,
+    )
+
+    for i in range(1):
+        test(model, id2token)
+
+
+def test(model, id2token):
+
+    start = time.time()
```

Flake8 (7.3.0)

[error] local variable 'start' is assigned to but never used

(F841)